src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -43,16 +49,35 @@ public List<LineResponse> findAllLines() {
 					.collect(Collectors.toList());
 	}
 
-	public LineResponse findLine(Long id) {
-		Optional<Line> line = lineRepository.findById(id);
+	public LineResponse findLine(Long lineId) {
+		Optional<Line> line = lineRepository.findById(lineId);
 		return LineResponse.of(line.get());
 	}
 
-	public void updateNameAndColor(Long id, LineRequest lineRequest) {
-		lineRepository.updateNameAndColor(lineRequest.getName(), lineRequest.getColor(), id);
+	@Transactional
+	public void updateNameAndColor(Long lineId, LineRequest lineRequest) {
+		Line line = lineRepository.getById(lineId);
+		line.updateNameAndColor(lineRequest.getName(), lineRequest.getColor());
 	}
 
+	@Transactional
 	public void deleteLine(Long id) {
 		lineRepository.deleteById(id);
 	}
+
+	@Transactional
+	public void addSection(Long lineId, SectionRequest sectionRequest) {
+		Optional<Station> upStation = stationRepository.findById(sectionRequest.getUpStationId());
+		Optional<Station> downStation = stationRepository.findById(sectionRequest.getDownStationId());
+		validationStaion(upStation, downStation);
```

Comment:
사소하지만 오타가 난 것 같아요 :)
```suggestion
		validationStation(upStation, downStation);
```

Reply:
앗 감사합니다!!